.github/workflows/ci.yaml

```diff
@@ -49,4 +49,10 @@ jobs:
     - name: Check GO format
       run: |
         go fmt -x ./...
-        git diff --exit-code || { echo 'Go sources need to be formated. Execute "go fmt -x ./..." locally and commit changes to fix an issue'; exit 1; }
+        git diff --exit-code || { echo 'Go sources need to be formatted. Execute "go fmt -x ./..." locally and commit changes to fix an issue'; exit 1; }
+
+    - name: Check GO license headers
+      run: |
+        pushd $(mktemp -d); go get -u github.com/google/addlicense; popd
+        addlicense --check -v -f license_header.txt $(find . -name '*.go' -not -path '*/vendor/*') \
+          || { echo 'Go sources are missing license headers'. Use addlicense to ensure license headers are included; exit 1; }
```

Comment:
`Use addlicense to ensure license headers are included` is not so specific to understand which command should be executed. What if we print the whole command here or even contribute Makefile with such command?

Reply:
Update message, but I don't think this is the reason to add a makefile when there's already a `build.sh` in the repo.

Reply:
Updated message clearly explain what to do to solve the issue. So, resolving this comment.

Just an example where Makefile and scripts could live together https://github.com/openshift/odo/blob/master/Makefile )